README.md

```diff
@@ -272,6 +274,27 @@ and test it:
 
 It should output the full ldif describing your site.
 
+#### Publishing both native OpenStack and ooi information
+
+In your `/var/lib/bdii/gip/provider/cloud-info-provider` include calls to both OpenStack and ooi
+providers:
+
+```sh
+#!/bin/sh
+
+cloud-info-provider-service --yaml /etc/cloud-info-provider/openstack.yaml \
+                            --middleware openstack \
+                            --os-username <username> --os-password <password> \
+                            --os-user-domain-name default --os-project-name <tenant> \
+                            --os-project-domain-name default --os-auth-url <auth-url>
+cloud-info-provider-service --yaml /etc/cloud-info-provider/openstack.yaml \
+                            --middleware ooi \
+                            --os-username <username> --os-password <password> \
+                            --os-user-domain-name default --os-project-name <tenant> \
+                            --os-project-domain-name default --os-auth-url <auth-url>
+```
+
+
```

Comment:
Probably one of the empty lines is not needed

Reply:
Fixed